DESCRIPTION

```diff
@@ -1,19 +1,23 @@
 Package: Siane
-Title: Siane
+Title: Explore the Spanish INE Data using this Geographic Tool
 Version: 0.1
-Description: Statistic polygon plots of INE data 
-Depends: R (>= 3.3.1)
-License: MIT
+Description: A tool that assists the user in two tasks: the map reading process and the
+  data representation process. The reading process is leveraged with functions that allow
+  the user to find maps in the Siane repository easily, i.e. searching by map year or
+  administrative level. The other main functionality is to bind numerical data to polygons.
+  This package is only compatible with the spanish IGN(National Institute of Geography) maps
+  and the INE(National Institute of Statistics) data.
+Depends: plyr, raster, rgdal, sp, methods, utils
```

Comment:
Packages in `Depends` are automatically loaded when the user loads your library. Does the user need them for using your package? If the answer is *no* for any of them, then they should be moved to `Imports` instead of `Depends`. See [Other dependencies](http://r-pkgs.had.co.nz/description.html) for further details about `Depends` and `Imports`.